mteb/models/colbert_models.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+import logging
+from collections.abc import Sequence
+from functools import partial
+from typing import Any
+
+import torch
+
+from mteb.model_meta import ModelMeta
+
+from .wrapper import Wrapper
+
+logger = logging.getLogger(__name__)
+
+
+class ColBERTWrapper(Wrapper):
+    def __init__(
+        self,
+        model_name: str,
+        **kwargs,
+    ) -> None:
+        """Wrapper for ColBERT models.
+
+        Args:
+            model_name: The ColBERT model to load from HuggingFace Hub.
+            **kwargs: Additional arguments to pass to the model.
+        """
+        try:
+            from pylate import models
+        except ModuleNotFoundError as e:
+            raise ModuleNotFoundError(
+                "To use the ColBERT models `pylate` is required. Please install it with `pip install mteb[pylate]`."
+            ) from e
+
+        self.model_name = model_name
+        self.static_model = models.ColBERT(self.model_name, **kwargs)
```

Comment:
I think it's better to rename as `model` or `colbert_model`

Reply:
Thanks a lot for the feedback! I’ve pushed the changes. Let me know if there’s anything else!